src/site-constants.js

```diff
@@ -7,8 +7,13 @@
 
 // NOTE: We can't just use `location.toString()` because when we are rendering
 // the SSR part in node.js we won't have a proper location.
+<<<<<<< HEAD
 const urlRoot = 'https://hu.reactjs.org';
 const version = '16.9.0';
+=======
+const urlRoot = 'https://reactjs.org';
```

Comment:
@tesseralis, @acdlite I am not sure what to do here. @gergely-nagy changed this to https://hu.reactjs.org. Who is right?

Reply:
It should be:

```
const urlRoot = 'https://hu.reactjs.org';
const version = '16.10.0';
```

The version was updated with the new release. the urlRoot should point to the website's url, which is different than the english version.